bedrock/mozorg/templates/mozorg/home/home-fr.html

```diff
@@ -100,7 +102,10 @@ <h1>{{ self.page_title() }}</h1>
         desc='Réalité virtuelle, IoT et plus encore.',
         link_cta='Découvrez le Web du futur',
         link_url='https://labs.mozilla.org/',
-        image_url='img/home/2018/billboard-open-minds.png',
+        image=resp_img('img/home/2018/billboard-open-minds.png',
+          srcset={
+            'img/home/2018/billboard-open-minds-high-res.png': '2x'
+        }),
         include_highres_image=True,
```

Comment:
Issue (blocking): looks like this is throwing an error on `/fr/` and needs removing
```suggestion
```